usr/libexec/greenboot/greenboot

```diff
@@ -7,28 +7,49 @@ SCRIPTS_CHECK_PATHS=("/usr/lib/greenboot/check" "/etc/greenboot/check")
 SCRIPTS_GREEN_PATHS=("/usr/lib/greenboot/green.d" "/etc/greenboot/green.d")
 SCRIPTS_RED_PATHS=("/usr/lib/greenboot/red.d" "/etc/greenboot/red.d")
 
+source_configuration_file() {
+    GREENBOOT_CONFIGURATION_FILE=/etc/greenboot/greenboot.conf
+    if test -f "$GREENBOOT_CONFIGURATION_FILE"; then
+	# shellcheck source=/etc/greenboot/greenboot.conf
+        source $GREENBOOT_CONFIGURATION_FILE
+    fi
+}
+
+source_configuration_file
+function is_disabled {
+    HEALTHCHECK=$1
+    for DISABLED_HEALTHCHECK in "${DISABLED_HEALTHCHECKS[@]}"; do
+        if [ "${HEALTHCHECK}" == "${DISABLED_HEALTHCHECK}" ]; then return 0; fi
+    done
+    return 1
+}
+
```

Comment:
nit for the two functions above: local variables should be not in uppercase to differentiate them from global variables